client/src/components/Search.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+class Search extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			search: '',
+			searchResult: []
+		}
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	// updates the search value
+	handleChange(e) {
```

Comment:
Dude! Nice work on this component!! 
My only suggestion would be... could you run Prettier over this file?

Reply:
ran prettier. Thanks!